index.html

```diff
@@ -0,0 +1,201 @@
+            the two major realms of Classical physics: Newtonian mechanics </p>
+        </div>
+       </div> 
+       <div class="user">
+        <div class="user__card">
+           <div ><img class="user__image" src="./img/fixed-height sec-.png" alt=""></div> 
+          <div class="user__desc">
+            <div><p class="user__name">Julian Jameson</p></div>
+            <div><p class="user__title">Profession</p></div>
+            <div  class="user__icons">
+            <i class="bi bi-instagram"></i>
+            <i class="bi bi-facebook"></i>
+            <i class="bi bi-twitter"></i>
+           </div>
+          </div>
+        </div>
+
+
+        <div class="user__card">
+           <div ><img class="user__image" src="./img/fixed-height sec2.png" alt=""></div> 
+          <div class="user__desc">
+            <div><p class="user__name">Julian Jameson</p></div>
+            <div><p class="user__title">Profession</p></div>
+            <div  class="user__icons">
+            <i class="bi bi-instagram"></i>
+            <i class="bi bi-facebook"></i>
+            <i class="bi bi-twitter"></i>
+           </div>
+          </div>
+        </div>
+
+
+         <div class="user__card">
+           <div ><img  class="user__image" src="./img/fixed-height sec-3.png" alt=""></div> 
+          <div class="user__desc">
+            <div><p class="user__name">Julian Jameson</p></div>
+            <div><p class="user__title">Profession</p></div>
+            <div  class="user__icons">
+            <i class="bi bi-instagram"></i>
+            <i class="bi bi-facebook"></i>
+            <i class="bi bi-twitter"></i>
+           </div>
+          </div>
+        </div>
+
+
+         <div class="user__card">
+           <div ><img class="user__image" src="./img/user-cover-4.png" alt=""></div> 
+          <div class="user__desc">
+            <div><p class="user__name">Julian Jameson</p></div>
+            <div><p class="user__title">Profession</p></div>
+            <div  class="user__icons">
+            <i class="bi bi-instagram"></i>
+            <i class="bi bi-facebook"></i>
+            <i class="bi bi-twitter"></i>
+           </div>
+          </div>
+        </div>
+       </div>
+  </main>
+    <!--footer  -->
+    <footer class="footer">
+        
+        <div class=" container__text contain__text--width">
+            <p class=" container__paragraph container__paragraph--color">newsletter</p>
+            <h2 class="container__heading">JOIN US</h2>
+            <p class="container__paragraph2">Problems trying to resolve the conflict between 
+            the two major realms of Classical physics: Newtonian mechanics </p>
+        </div>
+       <div> 
+        <div class="footer__input-container">
+            <input type="email" class="footer__input" placeholder="Your Email">
+            <button class="button__btn--primary button__btn--primary--radius">Subscribe</button>
+        </div>
+        
+      
+       
+    </footer>
+</body>
```

Comment:
I don't why you have so many `</body>` tags and we are supposed to have one of these please fix this and I can get a better view of what you have done

Reply:
those body tags were commented but i have fixed it